pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -254,14 +254,28 @@ func (r *ReconcileGrafanaDashboard) reconcileDashboards(request reconcile.Reques
 			}
 		}
 
-		status, err := grafanaClient.CreateOrUpdateDashboard(processed)
+		folder, err := grafanaClient.GetOrCreateNamespaceFolder(dashboard.Namespace)
+		if err != nil {
+			log.Info(fmt.Sprintf("failed to get or create namespace folder %v/%v with error %v", request.Namespace, request.Name, err))
```

Comment:
```suggestion
			log.Info(fmt.Sprintf("failed to get or create namespace folder %v with error %v", request.Namespace, err))
```

I _think_ this is a slightly more correct message, since the folder is just the namespace rather than namespace/name, right?

Reply:
Yes, correct the folder is titled after the namespace, the original thinking would be that if the function failed to get/create a folder we would know that it was for `dashboard X` from `namespace whatever`, but regardless, this definitely needs to be re-worded to make that info clearer.